backend/server.ts

```diff
@@ -260,7 +260,7 @@ app.get("*", (req, res) => {
 
 io.on("connection", socket => {
 	if (!onlineSessions.includes(socket.request.session.id)) onlineSessions.push(socket.request.session.id);
-	socket.emit("initialize", `Hello #${socket.request.session.id}`);
+	socket.emit("initialize", `Hello #${userList[socket.request.session.id] || socket.request.session.id}`);
```

Comment:
Please make a function in `backend/utils.ts` file which returns the username of the session id so all processes can use the same function to get the username and if something needs to be added in the future, it can be done easily.